Doc DuckDB have deprecated the usage of types in the sqllogictest (#2603)
duckdb/duckdb-web: docs/dev/sqllogictest/result_verification.md

10	20
```

> For legacy reasons the letters `R` and `T` are also accepted to denote columns. We have deprecated the usage of types in the SQLLogicTest. The DuckDB test runner does not use or need them internally.This means we only use I to denote columns.

Reviewer: ```suggestion
For legacy reasons the letters `R` and `T` are also accepted to denote columns.
> Deprecated DuckDB deprecated the usage of types in the SQLLogicTest. The DuckDB test runner does not use or need them internally – therefore, only `I` should be used to denote columns.

```
